### DISCUSSION TITLE
Pull requests not updated with commits pushed to source branch

### DISCUSSION THREAD

**Discussion by GuyLescalier** [community/community#69291] [Pull Requests]
### Select Topic Area

Bug

### Body

Hello,

We are using GitHub Server (on prem) 3.8.2 (upgarded to 3.8.10 hoping it would fix the issue but it didn't).

Since about a week or two, the pull requests are not being updated with new commits push by developers to the source branch.
Not sure if it's related to an other issue (#69289) since it started about the same time.

When we commit directly from github on the branch concerned by the PR, the PR is correctly updated.
Otherwise, we have to update it by editing the PR and resetting the same target branch.

Thanks for your help

**Comment by RudyB** [comment]
@GuyLescalier I am having the same issue with 3.10.0 and 3.11.0. I have also confirmed that commits made through the GitHub website directly do update the Pull Request however new commits pushed over https and ssh do not update the PR.

**Comment by RudyB** [comment]
@GuyLescalier I was able to resolve this problem with the assistance of github support. The solution was to [upload the certificate authority (CA) root certificates](https://docs.github.com/en/enterprise-server@3.11/admin/configuration/hardening-security-for-your-enterprise/troubleshooting-tls-errors#installing-self-signed-or-untrusted-certificate-authority-ca-root-certificates) to the github server.
The root cause of the problem was that the different github processes were unable to communicate with each other because the SSL certificate verification was failing.

**Comment by GuyLescalier** [comment]
@RudyB Thanks for sharing your solution, I'll try that.
In my case the problem stopped on it's own a while ago and is now reappearing so it's really wierd...
